specjbb/specjbb_run

```diff
@@ -219,7 +225,27 @@ set_prop_file()
 	fi
 	sed "s/input.ramp_up_seconds=30/input.ramp_up_seconds=${ramp_up}/g" $run_dir/SPECjbb.props > /tmp/temp
 	sed "s/input.measurement_seconds=240/input.measurement_seconds=${measurement}/g" /tmp/temp > /tmp/temp1
-	sed "s/#input.ending_number_warehouses=2/input.ending_number_warehouses=${end_warehouse}/g" /tmp/temp1 > /tmp/temp
+	if [ $jvms_doing -eq 0 ]; then
+		sed "s/#input.ending_number_warehouses=2/input.ending_number_warehouses=${end_warehouse}/g" /tmp/temp1 > /tmp/temp
+	else
+		value=`echo "$end_warehouse/$jvms_doing" | bc`
+		if [ $value -lt $start_warehouse ]; then
+			value=$start_warehouse
+		fi
+		sed "s/#input.ending_number_warehouses=2/input.ending_number_warehouses=${value}/g" /tmp/temp1 > /tmp/temp
+		tval=`echo "${increment_warehouse}/${jvms_doing}" | bc`
```

Comment:
It will match as close to possible the total thread count as possible as if we were running single jvm.  But understand what you are thinking.  Had to stew on it a bit, and then play around with it also.

Reply:
Verified the load points when running multiple jvms are the same between before and after the changes.

Before
Warehouses:Bops
32:2229897
64:3510866
96:3693642
128:3365315
160:2829296
192:2280709
224:1647832
256:1173676


(keep in mind, we are running in this case 4 times more then being reported)

After
32:2592416.43
64:4670087.55
96:5220099.51
128:5731682.54
160:5362906.05
192:5069989.14
224:4959569.14
256:4884733.87